src/coffee/product-type-import.coffee

```diff
@@ -75,7 +83,10 @@ class ProductTypeImporter
   ###
   import: (data) ->
     console.log "Importing product types to sphere"
-    Promise.map data.productTypes, (productType) ->
+    Promise.map data.productTypes, (productType) =>
+      console.log "Importing product type with name:", productType.name
```

Comment:
console is used in whole project .. for migrating from console to logger I would create another task